addons/binding/org.openhab.binding.gardena/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,38 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Gardena Binding
+Bundle-SymbolicName: org.openhab.binding.gardena;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.0.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-ClassPath: .
+Import-Package: 
+ com.fasterxml.jackson.annotation,
```

Comment:
puh ... that's a big change. I'll try it, but i don't know if GSON has all the required functions